Rubberduck.CodeAnalysis/Inspections/Concrete/EmptyMethodInspection.cs

```diff
@@ -0,0 +1,131 @@
+﻿using Rubberduck.Inspections.Abstract;
+using Rubberduck.Inspections.Results;
+using Rubberduck.Parsing.Common;
+using Rubberduck.Parsing.Inspections.Abstract;
+using Rubberduck.Resources.Inspections;
+using Rubberduck.Parsing.VBA;
+using System.Collections.Generic;
+using System.Linq;
+using Rubberduck.Parsing.Symbols;
+using Rubberduck.Resources.Experimentals;
+using static Rubberduck.Parsing.Grammar.VBAParser;
+
+namespace Rubberduck.Inspections.Concrete
+{
+
+    /// <summary>
+    /// Identifies empty module member blocks.
+    /// </summary>
+    /// <why>
+    /// Methods containing no executable statements are misleading as they appear to be doing something which they actually don't.
+    /// This might be the result of delaying the actual implementation for a later stage of development, and then forgetting all about that.
+    /// </why>
+    /// <example hasResults="true">
+    /// <![CDATA[
+    /// Sub Foo()
+    ///     ' ...
+    /// End Sub
+    /// ]]>
+    /// </example>
+    /// <example hasResults="false">
+    /// <![CDATA[
+    /// Sub Foo()
+    ///     MsgBox "?"
+    /// End Sub
+    /// ]]>
+    /// </example>
+    [Experimental(nameof(ExperimentalNames.EmptyBlockInspections))]
```

Comment:
I think it's fine to not have it marked as experimental; feel free to remove this attribute.